_config.yml

```diff
@@ -80,6 +80,10 @@ responsive_image:
   default_quality: 95
   strip: true
 
+# Compress Sass
+sass:
+  style: compressed
```

Comment:
Can you please give me a link that explains the benefit of this gem and using compressed?

Reply:
1. https://jekyllrb.com/docs/assets/
2. https://github.com/sass/sassc-ruby
3. Please see `sass` section on [this](https://cloudcannon.com/jekyll/2019/09/03/whats-new-jekyll-4/) page.